### DISCUSSION TITLE
About the 'fully download' message in the logs

### DISCUSSION THREAD

**Discussion by PH5HP** [rejetto/hfs#388] [General]
Hi,

Just a question about my observation. Some files have fully downloaded and some have not. Does that mean that the files without 'fully downloaded' are not downloaded by the user? See my screenshot below.

I know that some messages in the logs are not files downloaded from the content, but data send as part of the listing. The files in my screenshot without 'fully downloaded' are real files which as been requested as far as I can see in the log.

A cosmatic thing would widening the colomn with 'fully downloaded' with just 1 character.

Kind regards,
Henk - PH5HP - ftp.ph5hp.nl

![image](https://github.com/rejetto/hfs/assets/140949730/6aa057a8-ecee-4f24-a536-4974270911e0)

**Comment by rejetto** [comment]
> without 'fully downloaded' are not downloaded by the user? 

the point is not "by the user" but the "fully".
only requests where the last byte is sent have that caption added.
it may be not sent because interrupted early or because only first part of the file was requested (segmented download).

i tried windows, mac, chrome, firefox, safari, several resolutions, resizing the browser, but i wasn't able to reproduce your layout problem.
Any clue?
I surely can blindly add 5 pixels more, hoping for the best

**Reply by PH5HP** [reply]
Hi Rejetto,

What you are saying that somehow that files are not fully transferred to the user who requested it? Can you explain how 'segmented download' works in the browser? Is this a browser thing? Sorry for my ignorance on this topic.

Kind regards,
Henk - PH5HP - ftp.ph5hp.nl

**Comment by rejetto** [comment]
segmented download is normally not offered by browsers, just download managers.
if it happened with a browser, then it's surely the other case.
